sections/slideshow.liquid

```diff
@@ -155,17 +155,17 @@
         <div class="slideshow__text-wrapper banner__content banner__content--{{ block.settings.box_align }} page-width{% if block.settings.show_text_box == false %} banner--desktop-transparent{% endif %}">
           <div class="slideshow__text banner__box content-container content-container--full-width-mobile color-{{ block.settings.color_scheme }} gradient slideshow__text--{{ block.settings.text_alignment }} slideshow__text-mobile--{{ block.settings.text_alignment_mobile }}">
             {%- if block.settings.heading != blank -%}
-              <h2 class="banner__heading inline-richtext {{ block.settings.heading_size }}">
+              <h2 class="banner__heading inline-richtext {{ block.settings.heading_size }}{% if settings.animations_reveal_on_scroll %}{% if forloop.index == 1 %} scroll-trigger animate--slide-in{% endif %}{% endif %}" data-cascade>
```

Comment:
@melissaperreault, @kjellr in the Slideshow component, do we want each element to cascade in? Or just the text/button container to slide in?

Reply:
The text/button container as a whole is fine. 👍